front/app/containers/Admin/projects/project/permissions/index.tsx

```diff
@@ -19,18 +19,24 @@ import useFeatureFlag from 'hooks/useFeatureFlag';
 // style
 import styled from 'styled-components';
 import Outlet from 'components/Outlet';
+import { colors } from '../../../../../utils/styleUtils';
```

Comment:
Could we shorten this import to `import { colors } from 'utils/styleUtils';`?

Reply:
Yep, my editor always does this when it adds imports automatically